Add Hiring role definition (#17)
Vizzuality/devismos: _docs/backenders/strategy-2021/roles/hiring.md

---
layout: default
title: Template
parent: Roles
grand_parent: Strategy 2021
---

> # Hiring role definition -- *draft*

Reviewer: we should remove the "draft" and name the responsible person.

Reviewer 2: I think nobody wanted to take this role and somehow it ended up being weighted as lower priority.

As Agnieszka commented in the PR thread, we should either close this PR and shelve it, or keep it as draft with no responsible person. In practice I think I have been taking care (in a very lightweight fashion) of some of these tasks and in general I don't mind keep bringing up the "let's march on with hiring" mantra all the time, and except for some revisions to the job ad text the level of effort is normally negligible, so I'd even be happy to keep bugging the team with "let's march on with hiring" 🤷🏼

**Current responsible**: -

## Responsibilities

> - Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
>   
>     - Identified as responsibility to be shared with Capacity Role 
> 
> 
> - Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
>           
>     - Identified as responsibility to be shared with Capacity, Strategy, M.C roles 
>     

Reviewer 2: ```suggestion
- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
          
  *Responsibility to be shared with Capacity, Strategy and M.C roles*
```

Reviewer 2: what is the MC role though? 🤔

Author: Mentorship Coordination

    
- Distributing tasks: reviewing candidates, attending interviews

> 
> 

Reviewer 2: ```suggestion

```

- Liaising with P&C about onboarding of new BE members
  

>     - Identified as responsibility to be shared with Capacity, Strategy, M.C roles
>     

Reviewer 2: ```suggestion
    *Responsibility to be shared with Capacity, Strategy and M.C roles*
```

    
- Collaborating with P&C on job ads text
  

>     - Identified as responsibility to be shared with Capacity, Strategy roles
>     

Reviewer 2: ```suggestion
    *Responsibility to be shared with Capacity and Strategy roles*
```

> - Preparing candidate profile descriptions for outsourcing needs
> 
>    - Identified as responsibility to be shared with Capacity Role
> 
> 

Reviewer 2: ```suggestion
- Preparing candidate profile descriptions for outsourcing needs

  *Responsibility to be shared with Capacity role*
```

Reviewer 2: I think in practice these profiles will mostly be better handled by project developers with input from PMs, but I don't see any harm in keeping this here (especially if nobody is taking this role right now 😁)

---
layout: default
title: Template
parent: Roles
grand_parent: Strategy 2021
---

# Capacity role definition

**Current responsible**: -

***Note:***

> 2 people Identify the hiring related tasks to be managed between different roles:

Reviewer 3: Perhaps all these roles should be involved in the advice process on hiring. Might need adding to the "decision making" for these roles' descriptions. https://github.com/Vizzuality/devismos/pull/14 https://github.com/Vizzuality/devismos/pull/15 https://github.com/Vizzuality/devismos/pull/16

        
          Priority: Average Medium priority. (2 Low, 1 Medium, 1 High)
          Agn and Andr are happy with the way that Sebastion is managing his part of this responsibility
          Andr thinks this responsibility should me managed by each developer

- Liaising with P&C about onboarding of new BE members
       
          Priority: Medium (1 Low, 1 Medium, 2 High)
          Identified as responsability to be shared with Capacity, Strategy, M.C roles

- Collaborating with P&C on job ads text

          Priority: Medium-High ( 2 Medium, 2 High) 
          Identified as responsability to be shared with Capacity, Strategy roles
          Andr's note: Everyone should be encouraged to take part, with ownership on the Hiring and Strategy roles

- Preparing candidate profile descriptions for outsourcing needs
  
          Priority: Medium ( 3 Medium, 1 High) 
          Identified as responsability to be shared with Capacity Role
          Andr's note: This responsibility should me managed by each developer, with advice from at least another team member
                       We should try to keep this rather standard, once we can get there of course

## Impact

> 

Reviewer 3: Reduce administrative burden on the BE team
Improve timeliness of starting hiring processes

Author: Added 88912f620275e49546b134901c22eb14d9283e3f
Thanks 😃

---
layout: default
title: Template
parent: Roles
grand_parent: Strategy 2021
---

# Capacity role definition

**Current responsible**: -

***Note:***

2 people Identify the hiring related tasks to be managed between different roles:
- Capacity
- Strategy
- Mentorship Coordination

## Responsibilities

- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
          
          Priority: HIGH priority for all   
          Identified as responsibility to be shared with Capacity Role 

> - Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions 

Reviewer: is this repeated from line 23?

  
          Priority: HIGH priority for all   
          Identified as responsibility to be shared with Capacity, Strategy, M.C roles

- Distributing tasks: reviewing candidates, attending interviews
        
          Priority: Average Medium priority. (2 Low, 1 Medium, 1 High)

>           Agn and Andr are happy with the way that Sebastion is managing his part of this responsibility

Reviewer: keyboards are cheap, no need to save on characters typed :P please use full names

>           Andr thinks this responsibility should me managed by each developer

Reviewer 4: I don't think we should refer to BE members explicitly, this document should be independent of the current BE team members. I'd suggest phrasing these opinions ignoring who said them.
